### DISCUSSION TITLE
Is there a function to test for type, as in is_integer, is_string, is_map, is_list, ...?

### DISCUSSION THREAD

**Discussion by lutzbickhardt** [toitlang/toit#345] [Q&A] [answered]
I think I have seen it somewhere in the documentation, cannot find it again. (it would be helpful to have a reference of keyword (and built-in functions) in the doc.)

(is this the proper place to report things like this?)

**Comment by kasperl** [comment, marked as answer]
Yes, you can test using:

```
x is int
x is string
x is Map
x is List
```
